### PR TITLE
fix: critical fix on 0 being truncated when parsing zipcode

### PR DESCRIPTION
**Pull request by hurshd0**
# Description

What Changed ? e.g. Github Issues Title or Trello Card Title

**Please describe it here... in EDIT mode.**

## Type of change

**Please don't put a X mark here in EDIT mode**

- [ ] :recycle: :wastebasket: Re-factor, cleanup, un-comment, docstring
- [x] :ambulance: Bug fix (non-breaking change which fixes an issue)
- [ ] :sparkles: New feature (non-breaking change which adds functionality)
- [ ] :boom: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change Status

**Please don't put a X mark here in EDIT mode**

- [ ] :checkered_flag: Complete, tested, ready to review and merge
- [x] :traffic_light: Complete, but not tested (may need new tests)
- [ ] :construction: WIP work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

**Please don't put a X mark here in EDIT mode**

- [x] Manually Functionality Testing
- [ ] Unit Test

# Checklist

**Please don't put a X mark here in EDIT mode**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by ElijahMcKay — APPROVED** [review]
Looks good!  All edge cases now passing.

File: middleware/checkUsersMessageLimit.js [hunk 1]
```diff
@@ -19,41 +19,39 @@ function checkUsersMessageLimit(req, res, next) {
     req.get("origin") &&
     req.get("origin").includes(process.env.WEB_REQUEST_ORIGIN)
   ) {
-    postalCode = parseInt(req.body.zip);
+    postalCode = req.body.zip;
```
Comment (hurshd0) [inline middleware/checkUsersMessageLimit.js:22]: don't parse zipcodes to int

File: middleware/checkUsersMessageLimit.js [hunk 2]
```diff
@@ -19,41 +19,39 @@ function checkUsersMessageLimit(req, res, next) {
...
     phoneNumber = `+1${req.body.phone.replace(/[,.-]/g, "")}`;
 
-    // reassigning req.body to these values to use them in main route
-    req.body = {
-      postalCode: postalCode,
-      phoneNumber: phoneNumber,
-    };
   } else {
     // find out what the origin URL is from the Twilio Webhook to make this case more explicit
-    postalCode = parseInt(req.body.Body);
```
Comment (hurshd0) [inline middleware/checkUsersMessageLimit.js:32]: don't parse zipcodes to int
